### PR TITLE
synchronize-develop-with-master

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

File: src/core/shared/Notification.ts [hunk 1]
```diff
@@ -43,8 +41,9 @@ export class Notification {
               private _browserLocalStorage: BrowserLocalStorage,
               private _configProvider: ConfigProvider) {
     this._messageMap = new Map(Object.entries(Notification.NOTIFICATION_CLASSES));
-    this._notificationFrameElement = Notification.getNotificationContainer();
-
+    this._notificationFrameElement =
+      document.getElementById(this._configProvider.getConfig().componentIds.notificationFrame);
+    console.error(this._configProvider.getConfig().componentIds.notificationFrame);
```
Comment (wirus15) [inline src/core/shared/Notification.ts]: ?

Comment (pgs-gmartowski) [reply src/core/shared/Notification.ts]: 🤦‍♂ sorry, deleted

Comment (pgs-gmartowski) [reply src/core/shared/Notification.ts]: same in release2.0.7

**Comment by james-simpson-ST** [comment]
@pgs-gmartowski probably worth merging this but doing it again once 2.0.8 is merged to master

**Pull request closed without merging**